### PR TITLE
DOC: update query/eval figures on performance comparison 

### PR DESCRIPTION
**Pull request by carla-alves-24**
Feature reviewed by Joris (https://github.com/jorisvandenbossche) during the EuroScipy 2022 Sprint. 
This feature updates the pandas.eval() performance graphs here: https://pandas.pydata.org/docs/user_guide/enhancingperf.html#technical-minutia-regarding-expression-evaluation and the performance of the query here https://pandas.pydata.org/docs/user_guide/indexing.html#performance-of-query.

### CODE REVIEW

**Comment by phofl** [comment]
Hi, could you please fix the linting failures? You can also run pre-commit locally

**Comment by MatthieuLoustaunau** [comment]
hmm some tests are failing: 
```
FAILED pandas/tests/plotting/test_hist_method.py::TestSeriesPlots::test_hist_kde
FAILED pandas/tests/plotting/test_series.py::TestSeriesPlots::test_boxplot_series
FAILED pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_uhf - Ass...
FAILED pandas/tests/plotting/test_datetimelike.py::TestTSPlot::test_format_date_axis
FAILED pandas/tests/plotting/frame/test_frame.py::TestDataFramePlots::test_errorbar_with_integer_column_names
FAILED pandas/tests/plotting/frame/test_frame.py::TestDataFramePlots::test_errorbar_scatter
```
any clue about how to fix it ? @phofl

**Comment by phofl** [comment]
This is because of a matplotlib upgrade. Will be fixed if you merge main

**Comment by carla-alves-24** [comment]
> This pull request is stale because it has been open for thirty days with no activity. Please [update](https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#updating-your-pull-request) and respond to this comment if you're still interested in working on this.

**Comment by carla-alves-24** [comment]
I finished the work, I think you should verify to merge my commits. Thanks!

**Comment by jorisvandenbossche** [comment]
@carla-alves-24 my apologies for the slow responses. But thanks again for the contribution! 
Taking a final look right now

File: doc/source/user_guide/enhancingperf.rst [hunk 1]
```diff
@@ -820,17 +811,9 @@ significant performance benefit.  Here is a plot showing the running time of
 :func:`pandas.eval` as function of the size of the frame involved in the
 computation. The two lines are two different engines.
 
-
 .. image:: ../_static/eval-perf.png
```
Comment (jorisvandenbossche) [inline doc/source/user_guide/enhancingperf.rst:814]:
```suggestion
..
    The eval-perf.png figure below was generated with /doc/scripts/eval_performance.py

.. image:: ../_static/eval-perf.png
```

File: doc/source/user_guide/indexing.rst [hunk 2]
```diff
@@ -1440,13 +1451,13 @@ large frames.
 
 .. image:: ../_static/query-perf.png
```
Comment (jorisvandenbossche) [inline doc/source/user_guide/indexing.rst:1452]:
```suggestion
..
    The eval-perf.png figure below was generated with /doc/scripts/eval_performance.py

.. image:: ../_static/query-perf.png
```

**Review by jorisvandenbossche — APPROVED** [review]
The updates in the documentation look all good!

I will try to run the benchmark script later today locally as well, just to ensure the results on the figures we included now is somewhat reproducible across computers.

**Comment by jorisvandenbossche** [comment]
Did a local run of the script, looks all good! I basically got the same results (same conclusions). The line on the plot was a bit less noisy (I ensured to not have anything else running at the same time), so pushed the version of the figure that I obtained.

**Comment by carla-alves-24** [comment]
> @carla-alves-24 my apologies for the slow responses. But thanks again for the contribution! Taking a final look right now

@jorisvandenbossche thank you very much!

**Comment by jorisvandenbossche** [comment]
Thanks again!

**Pull request merged**